tests/unit/transformers/test_base.py

```diff
@@ -1,10 +1,374 @@
+from unittest.mock import patch
+
 import pandas as pd
+import pytest
 
 from rdt.transformers.base import BaseTransformer
 
 
 class TestBaseTransformer:
 
+    def test_get_subclasses(self):
+        """Test the ``get_subclasses`` method.
+
+        Setup:
+            -
+
+        Input:
+            -
+
+        Expected behavior:
+            -
+        """
+        pass  # TODO: not sure how to test this one.
+
+    def test_get_input_type(self):
+        """Test the ``get_input_type`` method.
+
+        Setup:
+            - mock the ``BaseTransformer``'s ``INPUT_TYPE`` attribute.
+
+        Output:
+            - the string stored in the ``INPUT_TYPE`` attribute.
+        """
+        with patch.object(BaseTransformer, 'INPUT_TYPE', 'categorical'):
+            # Run
+            input_type = BaseTransformer.get_input_type()
+
+            # Assert
+            expected = 'categorical'
+            assert input_type == expected
+
+    def test__add_prefix_none(self):
+        """Test the ``_add_prefix`` method when passed a ``None``.
+
+        The method should return ``None`` when ``None`` is passed as an argument.
+
+        Input:
+            - a None value.
+
+        Output:
+            - a None value.
+        """
+        # Setup
+        dictionary = None
+        base_transformer = BaseTransformer()
+
+        # Run
+        output = base_transformer._add_prefix(dictionary)
+
+        # Assert
+        expected = None
+        assert output == expected
```

Comment:
I do not thing a variable is necessary for these trivial cases. An `assert output is None` may be more readable.